src/main/java/core/basesyntax/db/InputFileReaderImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.db;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class InputFileReaderImpl implements InputFileReader {
+    @Override
+    public Scanner readFromFile(String filePath) {
+        Scanner scanner;
+        try {
+            scanner = new Scanner(new File(filePath));

```

Comment:
IMO a bit weird to just return a scanner, return the content , combine this with inputfilehandler

Reply:
OK. I thought, that it's a task condition to split these two processes.